core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DOAP.java

```diff
@@ -16,9 +16,22 @@
  */
 public class DOAP {
 
+	/**
+	 * The recommended prefix for the DOAP namespace: "doap"
+	 */
+	public static final String PREFIX = "doap";
+	
+	/**
+	 * The DOAP namespace: http://usefulinc.com/ns/doap#
+	 */
 	public static final String NAMESPACE = "http://usefulinc.com/ns/doap#";
 
 	/**
+	 * An immutable {@link Namespace} constant that represents the DOAP namespace.
+	 */
+	public static final Namespace NS = new SimpleNamespace(PREFIX, NAMESPACE);
```

Comment:
This doesn't compile: `SimpleNamespace` needs to be included in the imports of the `DOAP` class.